XRTK-Core/Packages/com.xrtk.core/Runtime/Providers/CameraSystem/BaseCameraDataProvider.cs

```diff
@@ -325,6 +334,7 @@ private bool SetupTrackingOrigin()
             // we can still keep going and assume everything is ready.
             var trackingOriginModeSet = true;
 
+#if !UNITY_EDITOR
```

Comment:
Why add this unnecessary preprocessor? The previous implemetnation was working perfectly fine without it?

Reply:
This will also break things when working with Meta Link or the mock device plugin.

Reply:
Shouldn't as it only affects the editor itself, but I think I see what you are getting at

Reply:
Removed